pkg/cloud/services/elb/loadbalancer.go

```diff
@@ -253,7 +253,7 @@ func (s *Service) getAPIServerClassicELBSpec() (*infrav1.ClassicELB, error) {
 			HealthyThreshold:   5,
 			UnhealthyThreshold: 3,
 		},
-		SecurityGroupIDs: []string{s.scope.SecurityGroups()[infrav1.SecurityGroupControlPlane].ID},
+		SecurityGroupIDs: []string{s.scope.SecurityGroups()[infrav1.SecurityGroupAPIServerLB].ID},
```

Comment:
Shouldn't this have both security groups?

Reply:
No, Traffic is allowed to the Control Plane members through an open egress policy on `SecurityGroupAPIServerLB` and the ingress policy allowing traffic to `SecurityGroupControlPlane` from this security group.

Reply:
In addition, because the ELB is externally facing with a public IP, they will receive NAT'd traffic from the control plane instances, which although technically continue to be in the datacentre appear to the ELB as internet originating traffic.

Different case for internal facing ELBs.

Reply:
Tested both external and internal ELB. With the current inbound rule 0.0.0.0/0 it doesn't matter if the traffic comes from internet originating traffic or internal, the security group rule allows access to the API server.